Multisig/UI/Assets/BalancesView.swift

```diff
@@ -8,12 +8,12 @@
 
 import SwiftUI
 
-struct CoinsView: View {
+struct BalancesView: View {
```

Comment:
The row height is too large

<img width="600" alt="Screenshot 2020-05-20 at 14 53 13" src="https://user-images.githubusercontent.com/284056/82448272-0606bc80-9a99-11ea-87f9-066eff74b749.png">

Reply:
done